README.md

```diff
@@ -5,13 +5,21 @@ The collection of machine learning algorithms that allows developers to stay wit
 Documentation will be available soon!  
 
 ## Installation and run
-- download and install [the Haskell Tool Stack](https://docs.haskellstack.org/en/stable/README/#how-to-install)  
+1. Make sure you have the required dependencies installed
+    - Dependencies:
```

Comment:
In the [hmatrix installation guide](https://github.com/haskell-numerics/hmatrix/blob/master/INSTALL.md) the [GSL](https://www.gnu.org/software/gsl/) was also mentioned
I searched this package on my machine and it wasn't installed, however there are no build errors -_-
Let's add this just in case
See next comments